tools/miq_secrets

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
+
+require "awesome_spawn"
+cmd = File.expand_path("decrypt_env_vars", __dir__)
+result = AwesomeSpawn.run!(cmd)
+
+require "yaml"
+env_vars = YAML.load(result.output)
+
+Process.detach(Kernel.spawn(env_vars, ARGV.join(" ")))
```

Comment:
Note we can switch this over to `AwesomeSpawn.run_detached` once that commit is included in a release